dotnet/test/common/NavigationTest.cs

```diff
@@ -115,8 +115,8 @@ public void ShouldRefreshPage()
         public Task ShouldNotHaveProblemNavigatingWithNoPagesBrowsedAsync()
```

Comment:
Then just `void` instead of `Task`?

Reply:
Good idea.

In NUnit 4, the async delegate assertions will be awaitable, so the tests will have `await Assert.ThatAsync(async ...)`. So this may be re-introduced in the future.

Should I change it now and change it back later, or keep it as is?

Reply:
Here please, we are living now. Insights: I don't know when NUnit 4 will be supported by bazel out of the box.